webroot/src/store/user/user.spec.ts

```diff
@@ -349,19 +355,9 @@ describe('User Store Actions', async () => {
         expect(commit.calledOnce).to.equal(true);
         expect(commit.firstCall.args).to.matchPattern([MutationTypes.SET_REFRESH_TIMEOUT_ID, null]);
     });
-    it('should successfully clear session stores', () => {
-        const dispatch = sinon.spy();
-
-        actions.clearSessionStores({ dispatch });
-
-        expect(dispatch.callCount).to.equal(5);
-    });
-    it('should successfully use the authType getter for licensee auth', () => {
+    it('should successfully clear update auth tokens by clearing existing ones except for auth token, then storing', () => {
```

Comment:
"should successfully **clear update auth** tokens..."  typo?

Reply:
Relatedly, if you're replacing the auth tokens why would you not want to clear the existing ones?

Reply:
We retain the access token from both user pools to maintain an indicator that we are still logged in to that user pool remotely. We need this indicator so that the app knows to remotely log the user out of all logged in user pools.

We clear all the other auth tokens (refresh token and auth type most importantly) to make the app function in an expected way (most recent logged in user is logged in, caused by changing auth_type). Clearing the refresh_token will _**most likely**_ revoke access of the initial user pool when the refresh window expires (the definition of logout to cognito). _**Most likely**_ is used above because within the first hour of login a user can revisit the log in page, pass through login using a previously set http only cookie without providing credentials so you aren't fully logged out.

As stated above, the only way to remove this pass through http only cookie and revoke the refresh is by visiting the logout url